LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 SEEK
```

Comment:
Nice catch. And this right here is why I have code reviews, even for simple changes 😄